src/components/Monaco/ReactMonacoEditor.tsx

```diff
@@ -24,6 +24,10 @@ const defaultOptions: monaco.editor.IStandaloneEditorConstructionOptions = {
   scrollBeyondLastLine: false,
   scrollbar: {
     alwaysConsumeMouseWheel: true,
+    verticalSliderSize: 4,
+    horizontalSliderSize: 4,
+    verticalScrollbarSize: 12,
```

Comment:
Adds "buffer" of 4px each side to match the Radix ScrollArea style and make it easier to click on when operated with mouse